cmd/dex/config.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/crypto/bcrypt"
 
+	"github.com/dexidp/dex/userinfo"
+	"github.com/dexidp/dex/userinfo/dai_drd"
```

Comment:
go-lang guidance says good package names are short and clear. They are lower case, with no under_scores or mixedCaps. We should use something like 'drdldap'

Reply:
agreed. will refactor